crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -80,6 +82,12 @@ pub struct EthTransactionRequest {
     /// EIP-2718 type
     #[cfg_attr(feature = "serde", serde(rename = "type"))]
     pub transaction_type: Option<U256>,
+    /// op-stack deposit source hash
+    pub source_hash: Option<H256>,
+    /// op-stack deposit mint
+    pub mint: Option<U256>,
+    /// op-stack deposit system tx
+    pub is_system_tx: Option<bool>,
```

Comment:
I would like to have these as a new container struct that is then used with `serde(flatten)` here
so just pub them in a `struct OptimismDepositRequestFields {}`
no strong feelings about naming